docs/technical_reference/property_models/activated_sludge_model.rst

```diff
@@ -0,0 +1,144 @@
+ "First density temperature coefficient", ":math:`A`", "dens_param_A", "None", ":math:`\text{K}^{-2}`"
+ "Second density temperature coefficient", ":math:`B`", "dens_param_B", "None", ":math:`\text{K}^{-1}`"
+ "Third density temperature coefficient", ":math:`C`", "dens_param_C", "None", ":math:`\text{dimensionless}`"
+ "First pressure coefficient", ":math:`\alpha`", "ref_pressure_correction", "None", ":math:`\text{dimensionless}`"
+ "Second pressure coefficient", ":math:`\beta`", "ref_pressure_slope", "None", ":math:`\text{Pa}^{-1}`"
+ "Reference viscosity (at 273 K)", ":math:`\mu_{ref}`", "mu_A", "None", ":math:`\text{kg/}\text{m/}\text{s}`"
+ "First viscosity temperature coefficient", ":math:`\mu_B`", "mu_B", "None", ":math:`\text{K}`"
+ "Second viscosity temperature coefficient", ":math:`\mu_C`", "mu_C", "None", ":math:`\text{K}`"
+
+**The parameters provided are valid between 0 and 350 °C and up to 600 bar.**
+
+
+Properties
+----------
+.. csv-table::
+  :header: "Description", "Symbol", "Variable", "Index", "Units"
+
+  "Component mass fraction", ":math:`x_j`", "mass_frac_phase_comp", "[p, j]", ":math:`\text{dimensionless}`"
+  "Mass density of fluid", ":math:`\rho`", "dens_mass_phase", "[p]", ":math:`\text{kg/}\text{m}^3`"
+  "Phase volumetric flowrate", ":math:`Q_p`", "flow_vol_phase", "[p]", ":math:`\text{m}^3\text{/s}`"
+  "Mass concentration", ":math:`C_j`", "conc_mass_phase_comp", "[p, j]", ":math:`\text{kg/}\text{m}^3`"
+  "Dynamic viscosity", ":math:`\mu`", "visc_d_phase", "[p]", ":math:`\text{Pa}\cdotp\text{s}`"
+  "Enthalpy flow", ":math:`H`", "enth_flow", "None", ":math:`\text{J/s}`"
+  "Soluble Inert Organic Matter", ":math:`S_I`", "inert_soluble", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Readily Biodegradable Substrate", ":math:`S_S`", "substrate", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate Inert Organic Matter", ":math:`X_I`", "inert_particulate", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Slowly Biodegradable Substrate", ":math:`X_S`", "biodegradable", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Active Heterotrophic Biomass", ":math:`X_{B,H}`", "heterotrophic", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Active Autotrophic Biomass", ":math:`X_{B,A}`", "autotrophic", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate Products Arising from Biomass Decay", ":math:`X_P`", "decay_particulate", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Oxygen", ":math:`S_O`", "oxygen", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Nitrate and Nitrite Nitrogen", ":math:`S_{NO}`", "nitrates", "[p]", ":math:`\text{kg/m}^{3}`"
+  "NH4 :math:`^{+}` + NH :math:`_{3}` Nitrogen", ":math:`S_{NH}`", "ammonium", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Soluble Biodegradable Organic Nitrogen", ":math:`S_{ND}`", "nitrogen_soluble", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Particulate Biodegradable Organic Nitrogen", ":math:`X_{ND}`", "nitrogen_particulate", "[p]", ":math:`\text{kg/m}^{3}`"
+  "Alkalinity", ":math:`S_{ALK}`", "alkalinity", "[p]", ":math:`\text{kg/m}^{3}`"
```

Comment:
These will need to be updated in accordance with the latest changes in PR #502 . Most of the names were altered for these. Be sure to look at both the thermophysical property model and the reaction prop model for ASM1.